.templates/remote-script.sh.template

```diff
@@ -30,7 +31,7 @@
 {% block after_script %}
     {% if remote_script == "docs" %}
     REMOTE_FAILED_FILE="tmp/nengo-fpga-$TRAVIS_JOB_NUMBER/{{ pkg }}/$TRAVIS_JOB_NUMBER.failed"
-    ssh -q {{ host }} [[ -e "$REMOTE_FAILED_FILE" ]] && scp {{ host }}:"$REMOTE_FAILED_FILE" .
+    exe ssh -q {{ host }} [[ -e "$REMOTE_FAILED_FILE" ]] && exe scp {{ host }}:"$REMOTE_FAILED_FILE" .
```

Comment:
oops. with the `exe`, if it doesn't find the `$REMOTE_FAILED_FILE`, travis-ci thinks the build is failing... hmmmm

Reply:
I think just having the `exe` on the `scp` portion should fix this? Because it shouldn't get to the `scp` part if it doesn't find the `REMOTE_FAILED_FILE`?